api/app/utils/review.inference.js

```diff
@@ -88,9 +88,9 @@ async function calculateWordDiff(data) {
       modifiedText = modifiedText.join(" ");
     }
     if (
-      (originalText.trim().startsWith("[") &&
-        originalText.trim().endsWith("]")) ||
-      (modifiedText.trim().startsWith("[") && modifiedText.trim().endsWith("]"))
+      (originalText.toString().trim().startsWith("[") &&
+        originalText.toString().trim().endsWith("]")) ||
+      (modifiedText.toString().trim().startsWith("[") && modifiedText.toString().trim().endsWith("]"))
     ) {
       let getOriginalText = originalText.trim().slice(1, -1);
```

Comment:
We should also add the `toString` method here. Same for getModifiedText
```suggestion
      let getOriginalText = originalText.toString().trim().slice(1, -1);
```